src/metrics.ts

```diff
@@ -157,7 +157,7 @@ export const MetricsProcessor = (
         },
         {
           key: TARGET_ATTRIBUTE,
-          value: event.target.identifier,
+          value: event?.target?.identifier ?? null,
```

Comment:
Just checking that `undefined` isn't allowed here and that it should either have a value or be `null`